WordCounter/Screen1/Controller/ViewController.swift

```diff
@@ -6,53 +6,37 @@
 //  Copyright © 2019 anhelina.mamekhina. All rights reserved.
 //
 
-import Foundation
 import UIKit
+import RxSwift
+import RxCocoa
 
-class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, TextProcessingDelegate {
+class ViewController: UIViewController {
     
-    var wordsCount: Dictionary<String, Int> = [:]
     let model = WordsCountModel()
+    let disposeBag = DisposeBag()
+    
     @IBOutlet weak var tableView: UITableView!
     
     override func viewDidLoad() {
         super.viewDidLoad()
         tableView.layer.cornerRadius = 10
-        
-        model.delegate = self
     }
     
     override func viewDidAppear(_ animated: Bool) {
         WaitManager.addSpinner(controller: self)
-        model.getDictOfWordsAndCounts()
+        tableView.dataSource = nil
+        tableView.delegate = nil
+        textProcessed()
     }
     
-    func textProcessed(textDictionary: Dictionary<String, Int>) {
-        wordsCount = textDictionary
-        tableView.reloadData()
+    func textProcessed() {
```

Comment:
The method corresponds to making reactive bindings. Rename it accordingly (i.e. `bindObservables` or `makeBindings`)